### DISCUSSION TITLE
custom icon light/dark version

### DISCUSSION THREAD

**Discussion by nembal** [retypeapp/retype#453] [Ideas]
team, loving retype and was just playing around with the icons. 

I found that there is "logodark" function in branding but there is likely no "icondark", right? 
Can we have that added? Would love to use my own icons supporting both modes.

**Comment by geoffreymcgill** [comment]
Can you provide an example where you would require an `icondark`?

**Reply by nembal** [reply]
Trying to make a Markdown code badge in `variant="contrast"`.

**Reply by geoffreymcgill** [reply]
Can you post a code sample demonstrating the scenario?

**Reply by nembal** [reply]
```md
[!badge variant="contrast" size="2xL" text="Medium"  text="Test Badge" iconAlign="right" logoDark="../static/stroke2.svg" icon="../static/stroke.svg"]
```

**Reply by nembal** [reply]
meant to use `icondark` not `logodark`
